cep-????.md

```diff
@@ -0,0 +1,160 @@
+
+### List of virtual packages
+
+In alphabetical order, every conda client MUST support the following virtual packages:
+
+- `__archspec`
+- `__cuda`
+- `__glibc`
+- `__linux`
+- `__osx`
+- `__unix`
+- `__win`
+
+#### `__archspec`
+
+This virtual package MUST be always present, with the version set to `1`. The build string SHOULD reflect the detected CPU microarchitecture when the target platform matches the native platform, 
+as provided by the generic values in the [`archspec/archspec-json` database](https://github.com/archspec/archspec-json/blob/v0.2.5/cpu/microarchitectures.json), plus some exceptions. For example,  `conda/conda` reports its generic values for Intel/AMD and `m*` names for Apple:
+
+- Generic: `x86_64_v1`, `x86_64_v2`, `x86_64_v3`, `x86_64_v4`, `arm`, `ppc`...
+- Apple: For M1, the value is `m1`. M2 and M3 are reported as `m2` and `m3`, respectively.
+
+If the microarchitecture cannot be detected or the target platform does not match the native platform, the build string MUST be set to the second component of the target platform, mapped with these rules:
+
+| Target platform | Reported `archspec` build string |
+|-----------------|----------------------------------|
+| `*-32` | `x86` |
+| `*-64` | `x86_64` |
+| `*-armv6l` | `armv6l`  |
+| `*-armv7l` | `armv7l`  |
+| `*-aarch64` | `aarch64`  |
+| `*-arm64` | `arm64`  |
+| `*-ppc64` | `ppc64`  |
+| `*-ppc64le` | `ppc64le`  |
+| `*-riscv64` | `riscv64`  |
+| `*-s390x` | `s390x`  |
+| `zos-z` | `0` |
+| Any other value | `0` |
+
+The build string MUST be overridable with the `CONDA_OVERRIDE_ARCHSPEC` environment variable, if set to a non-empty value.
+
+#### `__cuda`
+
+This virtual package MUST be present when the system exhibits GPU drivers compatible with the CUDA runtimes. When available, the version value MUST be set to the oldest CUDA version supported by the detected drivers (i.e. the formatted value of `libcuda.cuDriverGetVersion()`), constrained to the first two components (major and minor) and formatted as `{major}.{minor}`. The build string MUST be `0`.
+
+The version MUST be overridable with the `CONDA_OVERRIDE_CUDA` environment variable, if set to a non-empty value.
+
+#### `__glibc`
+
+This virtual package MUST be present when the native platform is `linux-*`. Its version value MUST be set to the system GNU `libc` version, constrained to the first two components (major and minor) formatted as `{major}.{minor}`. The build string MUST be `0`.
+
+The version MUST be overridable with the `CONDA_OVERRIDE_GLIBC` environment variable, if set to a non-empty value.
+
+If the GNU `libc` version could not be estimated (e.g. the tool is not running on Linux), the tool SHOULD provide a default value (e.g. `2.17`) and inform the user of that choice and its possible overrides; e.g. via `CONDA_OVERRIDE_GLIBC`, a CLI flag or a configuration file. The environment variable MUST be ignored when the target platform is not `linux~-*`.
```

Comment:
Should there be a difference between gnu libc not being present and the version not being parsable? I think conda should also be able to run on systems without glibc if packages dont require it.

Reply:
Maybe. What would be the expected behaviour? Something like this?

- Not found: `__glibc` does not show up.
- Not parsable: `__glibc=0` + warning/advice to override.

Reply:
I reworded this section a bit now so let's take another look.